doc/cookbook/simple_trees.rst

```diff
@@ -437,12 +437,61 @@ Branch lengths don't matter.
 
 .. jupyter-execute::
 
-    from cogent3 import load_tree
+    from cogent3 import make_tree
 
     tr1 = make_tree("(B:0.2,(C:0.2,D:0.2)F:0.2)G;")
     tr2 = make_tree("((C:0.1,D:0.1)F:0.1,B:0.1)G;")
     tr1.same_topology(tr2)
 
+Measure topological distances between two trees
+"""""""""""""""""""""""""""""""""""
```

Comment:
For rst, the number of heading characters (the `"` in this case) must be ≥ the length of the line above

Reply:
Resolved in 3813866